src/cmd-buildextend-live

```diff
@@ -71,6 +71,16 @@ if base_name == "rhcos" and args.fast:
 currently only available for FCOS")
     sys.exit(1)
 
+# scos: rename EFI/{distro} to EFI/centos
+# https://github.com/openshift/os/issues/954
+if base_name == "scos":
+    grubfilepath = ensure_glob(os.path.join(srcdir_prefix, 'EFI/*/grub.cfg'))
+    if len(grubfilepath) != 1:
+        raise Exception(f'Found != 1 grub.cfg files: {grubfilepath}')
+    # src/config/live/EFI/{redhat -> centos}
+    efipath = os.path.dirname(grubfilepath[0])
+    os.rename(efipath, os.path.dirname(efipath) + "/centos/")
```

Comment:
We should do this in the copy that's unpacked into `tmpimageefidir`, rather than modifying the source repository.

Reply:
Do you mean updating here https://github.com/coreos/coreos-assembler/blob/main/src/cmd-buildextend-live#L355?

Reply:
I think we want to rename the whole directory, so the second link looks better?

Reply:
Yes, after line 525.

Reply:
Thanks @cgwalters and @bgilbert, I will update after 525.